ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -164,6 +166,12 @@ export default function GasTiming({
           toHumanReadableTime(Number(customEstimatedTime?.upperTimeBound), t),
         ]);
       }
+    }
+    // code below needs to cleaned-up once EIP_1559_V2 flag is removed
+    else if (EIP_1559_V2) {
+      text = t('gasTimingNegative', [
+        toHumanReadableTime(low.maxWaitTimeEstimate, t),
```

Comment:
Do we want a test for this?

Reply:
The code has test coverage already, difference in V2 is infotip is removed from here.

Reply:
Where is the code the removes the info tooltip?

Reply:
Tooltip is moved next to link in V2 designs.

This `if` condition looks more clear to me